### ISSUE TITLE
Remove hardcoded dcos download URL.

### ISSUE THREAD

**Issue by AmalQB** [microservices-today/IaC-dcos#27]
There is a hardcoded DCOS download URL in 
https://github.com/microservices-today/IaC-dcos/blob/master/bootstrap.tf#L41
```
 provisioner "remote-exec" {
    inline = [
      "curl -O https://downloads.mesosphere.com/dcos/stable/dcos_generate_config.ee.sh",
      "rm -r $HOME/genconf; mkdir $HOME/genconf"
    ]
  }
```

**Comment by kamoljan** [comment]
+1

**Comment by alexraju** [comment]
Fixed with #31

**Comment by kamoljan** [comment]
thank you @alexraju !
